systers_portal/static/css/style.css

```diff
@@ -234,6 +235,20 @@ table.decoration-none  tr a {
   font-size: 1.3em;
 }
 
+/* Meetup Location CSS
+------------------------------------------------- */
+.box-boundary {
+  outline: 1px solid #eee;
+}
+
+.box-container {
```

Comment:
With the box-container you can do the following:
```css
.box-container {
  margin: 25px 0px;
  padding: 0px 25px;
  border: 1px solid #eee;
}
```
Removes the necessity to have an additional box for creating an border and manipulates the spacing using margin and padding.
By the way, do you know the difference between margin and padding?